spec/models/device_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe Device, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
Should we delete this spec file if we're not testing it?

Reply:
✅